src/main.rs

```diff
@@ -94,6 +96,25 @@ fn main() {
             app_args.isolate,
         );
 
+        if !app_args.no_context {
+            // Safe to unwrap: We just put this through an HTTP request
+            let mut clean_url = Url::parse(&final_url).unwrap();
+            clean_url.set_fragment(None);
+            // Safe to unwrap: must have a protocol and thus base 'cause we just used it.
```

Comment:
to match the style of the previous comment, I think that `// Safe to unwrap - must have a protocol and thus base because we just used it` would be more fitting